docs/build/program/apis/cloud.md

```diff
@@ -1332,3 +1332,262 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+
+- `user_id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The ID of the user to remove.
+
+**Returns:**
+
+- None.
+
+```python {class="line-numbers linkable-line-numbers"}
+member_list, invite_list = await cloud.list_organization_members()
+first_user_id = member_list[0].user_id
+
+await cloud.delete_organization_member(first_user_id)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.delete_organization_member).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### DeleteOrganizationInvite
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Delete a pending organization invite to the organization you are currently authenticated to.
+
+**Parameters:**
+
+- `email` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The email address the pending invite was sent to.
+
+**Returns:**
+
+- None.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if no pending invite is associated with the provided email address.
+
+```python {class="line-numbers linkable-line-numbers"}
+await cloud.delete_organization_invite("youremail@email.com")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.delete_organization_invite).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ResendOrganizationInvite
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Resend a pending organization invite email.
+
+**Parameters:**
+
+- `email` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The email address associated with the invite.
+
+**Returns:**
+
+- None.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if no pending invite is associated with the provided email address.
+
+```python {class="line-numbers linkable-line-numbers"}
+await cloud.resend_organization_invite("youremail@email.com")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.resend_organization_invite).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetRoverRentalRobots
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Return a list of rover rental robots within an {{< glossary_tooltip term_id="organization" text="organization" >}}.
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- [(List[viam.proto.app.RoverRentalRobot])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.RoverRentalRobot): The list of rover rental robots.
+
+```python {class="line-numbers linkable-line-numbers"}
+rental_robots = await cloud.get_rover_rental_robots()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_rover_rental_robots).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### CheckPermissions
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Check validity of a list of permissions.
```

Comment:
I assume this means they are authorized for a resource? @stuqdog Can you confirm

Reply:
Added that below

Reply:
Yeah so `check_permissions` takes a list of permissions and returns a filtered list of which ones are actual, so e.g., if you have permission to `control_robot` and you pass a `control_robot` permission then you will get that permission back. If you _don't_ have `control_robot` permission and you pass it as your argument, you will get nothing back.